sparsehash/dense_hash_map

```diff
@@ -105,6 +105,17 @@
 #include <sparsehash/internal/densehashtable.h>  // IWYU pragma: export
 #include <sparsehash/internal/libc_allocator_with_realloc.h>
 
+template <class T>
+struct remove_cvref {
+  typedef typename std::remove_cv<typename std::remove_reference<T>::type>::type type;
```

Comment:
maybe https://en.cppreference.com/w/cpp/types/decay could be useful

Reply:
@Dekken Thanks! I changed it to using `std::decay`

Reply:
you can remove my commit, just squash with yours as the real one and I'll merge

thanks

Reply:
Okay